server/libs/aws/batch.js

```diff
@@ -87,6 +113,61 @@ export default aws => {
       )
     },
 
+    /*
+     * Looks for rogue jobs and terminates them as needed
+     */
+    _cleanupJobs() {
+      // Subtract two days from the current date to search
+      const twoDaysAgo = new Date()
+      twoDaysAgo.setDate(twoDaysAgo.getDate() - 2)
+      // This query will get some jobs we don't want to stop since
+      // those jobs may have spent time in pending or runnable
+      // Tasks are filtered to only long run times in _terminateOldJobs
+      const query = {
+        'analysis.status': 'RUNNING',
+        'analysis.started': { $lte: twoDaysAgo },
```

Comment:
I am not seeing a 'started' property in my local db.  Is this is supposed to be 'created'?